conda-forge.yml

```diff
@@ -9,3 +9,11 @@ github:
 conda_build:
   error_overlinking: true
 conda_forge_output_validation: true
+github_actions:
+  timeout_minutes: 2160
```

Comment:
36hours?! Let's see how long the builds actually take, and maybe we should consider moving to a larger CPU runner like `cirun-openstack-cpu-xlarge` with more RAM, and disable the `cirun-openstack-gpu-large` GPU runner because [OpenStack has a 4GPU server-wide limit](https://github.com/Quansight/open-gpu-server?tab=readme-ov-file#limitations) and we don't want to hog these machines for that long.

Reply:
Yes. The job timed out at 6 hours with about 20% completed. Linear projection yields 30 hours. Bumped it up to 36 hours because it's a multiple of 12.